src/Microsoft.Azure.WebJobs.Host/Executors/FunctionInstance.cs

```diff
@@ -51,6 +52,18 @@ public IFunctionInvoker Invoker
             get { return _invoker; }
         }
 
+        public object JobInstance
+        {
+            get
+            {
+                if (_jobInstance == null)
+                {
+                    _jobInstance = Invoker.CreateInstance();
```

Comment:
Actually, maybe just remove this whole property and put JobInstance on the ParameterHelper (where the other instantiated arguments live). That will solve of these issues. 

---
In reply to: [132730667](https://github.com/Azure/azure-webjobs-sdk/pull/1292#discussion_r132730667) [](ancestors = 132730667)